cli/flox/src/commands/install.rs

```diff
@@ -480,21 +480,24 @@ fn prompt_to_modify_rc_file() -> Result<bool, anyhow::Error> {
         Shell::Tcsh(_) => r#"eval "`flox activate -d ~ -m run`""#,
         Shell::Fish(_) => "flox activate -d ~ -m run | source",
     };
-    let rc_file_name = match shell {
-        Shell::Bash(_) => ".bashrc",
-        Shell::Zsh(_) => ".zshrc",
-        Shell::Tcsh(_) => ".tcshrc",
-        Shell::Fish(_) => "config.fish",
+    let rc_file_names = match shell {
+        Shell::Bash(_) => vec![".bashrc", ".profile"],
+        Shell::Zsh(_) => vec![".zshrc"],
```

Comment:
blocking question: did you confirm we don't need to add another file for zsh? And I guess tcsh

Reply:
`tcsh` reads `~/.tcshrc` in both login and non-login shells. Zsh we should probably also put something in `~/.zshenv`.